benchmark/src/main/content/bin/kc-chaos.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Randomly delete pods in a Kubernetes namespace.
+set -e
+
+: ${INITIAL_DELAY_SECS:=30}
+: ${CHAOS_DELAY_SECS:=60}
+: ${PROJECT:="runner-keycloak"}
+
+sleep $INITIAL_DELAY_SECS
+echo -e "\033[0;31m INFO: STARTING CHAOS SCENARIO - DELETING RANDOM KC POD with a CHAOS_DELAY_SECS of $CHAOS_DELAY_SECS"
```

Comment:
It would be nice if the script would output a line just after the start, and then do the initial sleep, as it provides IMHO a better user experience as it shows the user that something is happening / about to happen.